spec/display-buffer-spec.coffee

```diff
@@ -703,6 +715,11 @@ describe "DisplayBuffer", ->
       expect(displayBuffer.screenPositionForBufferPosition([100000, 0])).toEqual [12, 2]
       expect(displayBuffer.screenPositionForBufferPosition([100000, 100000])).toEqual [12, 2]
 
+    it "clips correctly around atomic tab characters", ->
+      buffer.insert([0, 0], '\t')
```

Comment:
Correct, however if you leave `tabLength` as it is (namely, `= 2`) spec will pass anyway, even if we remove `+ token.bufferDelta`. That's because with `2`, if you execute:

```coffee
displayBuffer.screenPositionForBufferPosition([0, 0])
```

It will try to clip `1`, which is exactly in the middle and `closest` will always prefer to clip to `0` (behaving exactly as `left`). We need a higher value, such as `4`, which will try to clip to something past the middle (e.g. `3`), reporting an error if we leave out `+ token.bufferDelta`.